user_guide_src/source/changelogs/v4.5.2.rst

```diff
@@ -14,6 +14,12 @@ Release Date: Unreleased
 BREAKING
 ********
 
+************
+Enhancements
+************
+- Added `retainMultiplePatterns()` to `FileCollection` class.
+
+
```

Comment:
This change must go into https://github.com/codeigniter4/CodeIgniter4/blob/4.6/user_guide_src/source/changelogs/v4.6.0.rst